components/sysutils/virtualbox/Makefile

```diff
@@ -14,30 +14,39 @@
 # Copyright 2019 Tim Mooney
 # Copyright 2020 Jim Klimov
 # Copyright 2021 omnios.org wrt the ctx patches
+# Copyright 2022 Daniel Bell
 
 #USE_OPENSSL11= yes
 include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=         VirtualBox
-COMPONENT_VERSION=      6.1.36
+COMPONENT_VERSION=      6.1.38
 COMPONENT_REVISION =	0
 COMPONENT_SUMMARY=      VirtualBox - general-purpose full virtualizer
 COMPONENT_PROJECT_URL=  https://www.virtualbox.org/
 COMPONENT_FMRI=         system/virtualbox
 COMPONENT_CLASSIFICATION= System/Virtualization
+
+COMPONENT_DOWNLOAD_URL= https://download.virtualbox.org/virtualbox/$(COMPONENT_VERSION)
 COMPONENT_SRC=			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+
+# Grab checksum file
+$(shell cd $(USERLAND_ARCHIVES) && [ -f $(COMPONENT_SRC).sums ] || curl -o $(COMPONENT_SRC).sums https://www.virtualbox.org/download/hashes/$(COMPONENT_VERSION)/SHA256SUMS )
```

Comment:
This is nice way how to detect new values for `SDK_BUILD_NUMBER` and `COMPONENT_ARCHIVE_HASH`, but I believe we still need the actual values recorded here in the `Makefile`.  This would help to detect cases when upstream replace the source tarball with some newer/better/fixed version.  If we won't detect that we could get surprising results.

Thanks.